app/lib/donations/controllers/DonorsChooseDonationController.js

```diff
@@ -403,12 +407,12 @@ DonorsChooseDonationController.prototype.submitDonation = function(donorInfoObje
             sendSuccessMessages(donorPhone, donationConfig, jsonBody.proposalURL);
           }
           else {
-            logger.warn('DonorsChoose.requestDonation failed for user:' + donorPhone + ' proposal:' + proposalId + ' body:', jsonBody);
+            logger.warn('DonorsChoose.requestDonation status!=success user:' + donorPhone + ' proposal:' + proposalId + ' body:', jsonBody);
             sendSMS(donorPhone, donationConfig.error_start_again);
           }
         }
         catch (e) {
-          logger.error('DonorsChoose.requestDonation error for user:%s error:%s', donorPhone, e.message);
+          logger.error('DonorsChoose.requestDonation catch exception for user:%s e:%s', donorPhone, e.message);
```

Comment:
This is the message I reported in https://github.com/DoSomething/ds-mdata-responder/pull/553#issuecomment-237643864.

My Procfile (didn't commit yet) is set to `web: node --max_old_space_size=2000 server.js` and I can see that I get the `DonorsChoose.requestDonation success` message, but the exception is thrown in either `updateUserWithDonation` or `sendSuccessMessages`:

````
          if (jsonBody.statusDescription == 'success') {
            logger.info('DonorsChoose.requestDonation success for user:' + donorPhone + ' proposal:' + proposalId + ' body:', jsonBody);
            updateUserWithDonation();
            sendSuccessMessages(donorPhone, donationConfig, jsonBody.proposalURL);
          }
````